src/raft/leader.go

```diff
@@ -320,22 +327,20 @@ func (r *Leader) processHeartbeatResponse(ackGranted *int, rsp *model.RaftRPCRes
 		}
 
 		// to reset nextPuregeBinlog:
-		// we must get all responses from the follower(s)
+		// we must get all responses from the follower(s) and idle(s)
 		// imagine that:
 		// Master is doing backup for Slave2 restore
 		// Master purged to Slave1-Relay_Master_Log_File
 		// Slave2 starts up and can't find the binlog which she is want
-		if *ackGranted == r.getMembers() {
+		if *ackGranted == r.getAllMembers() {
```

Comment:
Normally If judgment is invalid in this code. Because https://github.com/radondb/xenon/pull/82/commits/a8181bb6399353745475a3c86f825e555ffd1014#diff-2f3d922f6eeea47882dafedefd3a1025R319